README.md

```diff
@@ -1,75 +1,226 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+<a name="readme-top"></a>
 
-# Project Name
+<div align="center">
 
-> Description the project.
+  <img src="soyMarco.png" alt="logo" width="140"  height="auto" />
+  <br/>
 
+  <h3><b>Awesome Books ES6 | README</b></h3>
 
-## Built With
+</div>
 
-- Major languages
-- Frameworks
-- Technologies used
+<!-- TABLE OF CONTENTS -->
 
-## Live Demo (if available)
+# 📗 Table of Contents
 
-[Live Demo Link](https://livedemo.com)
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
+<!-- PROJECT DESCRIPTION -->
 
-## Getting Started
+# 📖 Capstone Project #1 <a name="about-project"></a>
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+<div align="center">
 
+  <img src="web_mockup.png" alt="logo" width="450"  height="auto" />
```

Comment:
- [ ] The logo is broken, kindly update it
![image](https://user-images.githubusercontent.com/62705947/228285621-08c56f7e-1ecf-49e3-95bd-41628960c12e.png)